pkg/sshd/config.go

```diff
@@ -1,9 +1,18 @@
 package sshd
 
+import (
+	"time"
+)
+
 // Config represents the required SSH server configuration
 type Config struct {
-	SSHHostIP                  string `envconfig:"SSH_HOST_IP" default:"0.0.0.0" required:"true"`
-	SSHHostPort                int    `envconfig:"SSH_HOST_PORT" default:"2223" required:"true"`
-	HealthSrvPort              int    `envconfig:"HEALTH_SERVER_PORT" default:"8092"`
-	HealthSrvTestStorageRegion string `envconfig:"HELATH_SERVER_TEST_STORAGE_REGION" default:"us-east-1"`
+	SSHHostIP                   string `envconfig:"SSH_HOST_IP" default:"0.0.0.0" required:"true"`
+	SSHHostPort                 int    `envconfig:"SSH_HOST_PORT" default:"2223" required:"true"`
+	HealthSrvPort               int    `envconfig:"HEALTH_SERVER_PORT" default:"8092"`
+	HealthSrvTestStorageRegion  string `envconfig:"HEALTH_SERVER_TEST_STORAGE_REGION" default:"us-east-1"`
+	CleanerPollSleepDurationSec int    `envconfig:"CLEANER_POLL_SLEEP_DURATION_SEC" default:"1"`
```

Comment:
the only semantic addition here is `CleanerPollSleepDurationSec`, but the spacing of the types changed